examples/fleet/kuard-blue-green/README.md

```diff
@@ -0,0 +1,118 @@
+   ```
+
+### 2. Deploy the demo application to the member clusters
+
+1. Switch to `aks-member-blue` context to create deployment using kuard:blue,
+
+   ```shell
+   KUBECONFIG=member-blue kubectl create namespace net
+
+   KUBECONFIG=member-blue kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard-blue-green/kuard-blue.yaml
+   ```
+
+2. Switch to the `aks-member-green` context to create deployment using kuard:green,
+
+   ```shell
+   KUBECONFIG=member-green kubectl create namespace net
+
+   KUBECONFIG=member-green kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard-blue-green/kuard-green.yaml
+   ```
+
+### 3. Create service and service export in the member clusters
+
+1. Switch to `aks-member-blue` context to create the kuard service with an internal load balancer and service export:
+
+   ```shell
+   KUBECONFIG=member-blue kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard-blue-green/kuard-export.yaml
+   ```
+
+2. Verify the service is created:
+
+   ```shell
+   KUBECONFIG=member-blue kubectl get service -n net
+   ```
+
+    The output is similar to:
+
+    ```console
+    NAME    TYPE           CLUSTER-IP   EXTERNAL-IP   PORT(S)          AGE
+    kuard   LoadBalancer   10.0.107.1   <none>        8080:32450/TCP   78s
+    ```
+3. Verify the service export is valid and has no conflicts:
+
+   ```shell
+   KUBECONFIG=member-blue kubectl get serviceexport -n net
+   ```
+
+   The output is similar to:
+
+    ```console
+    NAME    IS-VALID   IS-CONFLICTED   AGE
+    kuard   True       False           31s
+    ```
+4. Switch to `aks-member-green` context to create the kuard service with an internal load balancer and service export:   
+
+   ```shell
+   KUBECONFIG=member-green kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard-blue-green/kuard-export.yaml
+   ```
+   
+   Verify the service and service export following the instructions above. 
+
+### 4. Expose fleet-wide endpoints from exported services with a multi-cluster service
+
+1. Switch to `aks-member-green` context to create a multi-cluster service. The endpoints will then be exposed with a L4 load balancer using `aks-member-green` nodes.
+
+   ```shell
+   KUBECONFIG=member-green kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/kuard-blue-green/kuard-mcs.yaml
+   ```
+2. Verify the mutli-cluster service is valid
+
+    ```shell
+   KUBECONFIG=member-green kubectl get mcs -n net
+   ```
+
+   The output is similar to:
+
+   ```console
+   NAME    SERVICE-IMPORT   EXTERNAL-IP   IS-VALID   AGE
+   kuard   kuard            20.81.3.74    True       26s
+   ```
+3. Access the running service:
+
+   ```shell
+   curl 20.81.3.74:8080
+   ```
+   Or access the http://20.81.3.74:8080 in the browser.
+
+   The request will randomly hit different hostnames with blue or green version.

```

Comment:
is this going to be proportional to the number of pods in each deployment? (I know it is :) )